src/main/webapp/src/components/songInfo.js

```diff
@@ -144,6 +146,32 @@ class SongInfo extends React.Component {
       ],
     };
 
+    function drawChart() {
+      const data = new google.visualization.DataTable();
+      data.addColumn('string', 'Name');
+      data.addColumn('number', 'Salience');
+      songInfo.entityAnalysis.forEach((entity) => {
+        data.addRow([entity.word, entity.salience]);
+      });
+
+      const options = {
+        title : 'Most important words',
```

Comment:
remove space between `title` and `:`. Same for other fields.

Reply:
Spaces are my enemy.